src/auth/ui/vue/authForms/manageIdentityCenter.vue

```diff
@@ -155,25 +181,50 @@ export default defineComponent({
             this.emitAuthConnectionUpdated({ id: this.state.id, isConnected: actualIsConnected, cause })
         },
         async getRegion() {
+            client.startAuthFormInteraction(this.state.featureType, 'iamIdentityCenter')
             const region = await this.state.getRegion()
+            if (!region) {
+                return
+            }
+            this.errors.region = ''
```

Comment:
In what situation would there ever be a region error? Shouldn't it default to something?

Reply:
This is for errors like "please select a region"